modules/nw-multus-bridge-object.adoc

```diff
@@ -1,25 +1,34 @@
 // Module included in the following assemblies:
 //
 // * networking/multiple_networks/configuring-additional-network.adoc
+// * microshift_networking/microshift_multiple_networks/microshift_cni_multus.adoc
+
 :_mod-docs-content-type: REFERENCE
 [id="nw-multus-bridge-object_{context}"]
 = Configuration for a bridge additional network
 
-The following object describes the configuration parameters for the bridge CNI
-plugin:
+The following object describes the configuration parameters for the Bridge CNI plugin:
 
 .Bridge CNI plugin JSON configuration object
 [cols=".^2,.^2,.^6",options="header"]
 |====
 |Field|Type|Description
 
+ifndef::microshift[]
 |`cniVersion`
 |`string`
 |The CNI specification version. The `0.3.1` value is required.
 
 |`name`
 |`string`
 |The value for the `name` parameter you provided previously for the CNO configuration.
+endif::microshift[]
+
```

Comment:
Do we need to use the ifdefs for `cniVersion`? It looks the same to me in both versions.